src/features/line_numbers.rs

```diff
@@ -7,6 +7,8 @@ use crate::config;
 use crate::delta::State;
 use crate::features::hyperlinks;
 use crate::features::side_by_side;
+use crate::features::side_by_side::LeftRight;
+use crate::features::side_by_side::PanelSide::{Left, Right};
```

Comment:
Since `Left` and `Right` and `LeftRight` are used here in `line_numbers.rs`, I think that raises the questions:

1. Should they be called `Minus` and `Plus` rather than `Left` and `Right`? (because they are used in contexts other than side-by-side, and because the rest of the codebase uses "minus" and plus" with this meaning)
2. Should they be defined somewhere more central than `side-by-side.rs`?

However, even if those questions are reasonable, I am happy to leave them unresolved for now and let you decide in the main side-by-side PR.